manifests/params.pp

```diff
@@ -26,4 +26,5 @@
   Stdlib::HTTPSUrl $candlepin_url = "https://${candlepin_host}:${candlepin_port}/candlepin",
   String[1] $candlepin_client_keypair_group = 'foreman',
   String[1] $postgresql_evr_package = $katello::globals::postgresql_evr_package,
+  String[0] $meta_package = 'katello',
```

Comment:
If we make it `String[0]`, I'd prefer to also implement the empty string as "don't install it". Otherwise make it `String[1]`.